apps/passport-server/test/frogcrypto.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { expect } from "chai";
+import MockDate from "mockdate";
+import { Pool } from "postgres-pool";
+import { EdDSAFrogPCD, EdDSAFrogPCDPackage } from "@pcd/eddsa-frog-pcd";
+import {
+  createFeedCredentialPayload,
+  pollFeed,
+  PollFeedResult,
+  requestListFeeds
+} from "@pcd/passport-interface";
+import { AppendToFolderAction, PCDActionType } from "@pcd/pcd-collection";
+import { Identity } from "@semaphore-protocol/identity";
+import { stopApplication } from "../src/application";
+import { getDB } from "../src/database/postgresPool";
+import { Zupass } from "../src/types";
+import {
+  FROGCRYPTO_FEEDS,
+  FrogCryptoFeed,
+  FrogCryptoFeedConfig
+} from "../src/util/frogcrypto";
+import { goodResponse } from "./tripsha/mockTripshaAPI";
+import { testLogin } from "./user/testLoginPCDPass";
+import { overrideEnvironment, testingEnv } from "./util/env";
+import { startTestingApp } from "./util/startTestingApplication";
+import "mocha";
+
+describe("frogcrypto functionality", function () {
+  this.timeout(30_000);
+  let db: Pool;
+  let application: Zupass;
+  let identity: Identity;
+  let frogPCD: EdDSAFrogPCD;
+
+  this.beforeAll(async () => {
+    await overrideEnvironment(testingEnv);
+    db = await getDB();
+
+    application = await startTestingApp();
+
+    await EdDSAFrogPCDPackage.init?.({});
+  });
+
+  this.afterAll(async () => {
+    await stopApplication(application);
+    await db.end();
+  });
+
+  it("should be able to log in", async function () {
+    const result = await testLogin(application, goodResponse.tickets[0].email, {
```

Comment:
```suggestion
    const result = await testLogin(application, randomEmail(), {
```

want to avoid unnecessary dependencies between tests. you're going to have to import `randomEmail` too for this suggestion to work.